packages/stryker-elements/package.json

```diff
@@ -33,7 +33,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "lint:fix": "eslint --ext .ts,.js src --fix"
```

Comment:
I suppose this could also go in the root of the project, but personally I just use `npx eslint . --fix` for this purpose. Not bad to have it though.

Reply:
Resolving = keep it as it is? 🫠

Reply:
Do you want it moved up to the root package scripts?

Reply:
Fixed in 26c22ddf47f42d8f45987911de8989e69dd5408c